# Test cases automatically generated by Pynguin (https://www.pynguin.eu).
# Please check them before you use them.
import pytest
import max_leet_1 as module_0


# @pytest.mark.xfail(strict=True)
def test_case_0(): # FAILED - IndexError: list index out of range
    bool_0 = False
    list_0 = [bool_0, bool_0, bool_0]
    int_0 = -716
    solution_0 = module_0.Solution()
    solution_0.findMaxAverage(list_0, int_0)


def test_case_1(): # PASSED
    solution_0 = module_0.Solution()
    bool_0 = True
    list_0 = [bool_0, bool_0, bool_0, bool_0]
    bool_1 = True
    float_0 = solution_0.findMaxAverage(list_0, bool_1)
    assert float_0 == pytest.approx(1.0, abs=0.01, rel=0.01)


# @pytest.mark.xfail(strict=True)
def test_case_2(): # FAILED - TypeError: object of type 'bool' has no len()
    bool_0 = True
    solution_0 = module_0.Solution()
    solution_0.findMaxAverage(bool_0, bool_0)
